test/functional/cmdLineTests/lockWordAlignment/build.xml

```diff
@@ -73,7 +73,9 @@
 
 	<target name="generateTestClass" depends="compile" description="Generate test class files ">
```

Comment:
This target needs to be specific to Java 11, the --add-exports won't work for Java 8.
Also the asm-all.jar can be removed from the path since it's not used. Here and for the compile target above.

Reply:
--add-exports have been set only for 11+, and asm-all has been removed from both targets. Thanks.